### PR TITLE
document reference to embedded design migration

### PR DESCRIPTION
**Pull request by gabrielhase**

### CODE REVIEW

File: guides/migrate-to-embedded-design.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+The main benefit for customers is that they can completely separate Livingdocs designs from whatever technology they want to use in the frontend. E.g. Swisscom has embedded designs in the server project and uses the HTML framework nitro in the frontend (without any dependency on Livingdocs).
+
+## Overview
+
+Prerequesites: you need to have a configurable channel, meaning that your Livingdocs configuration is done via a seeding process of project configuration files in your server repository and **not via static configuration files**.
+
+In order to go from a reference design to an embedded design the following steps are necessary:
+- migrate the design config from v1 to v2 (currently, all reference designs we know of are v1)
+- move all design specific files (design_settings and all components) to the server repository (and out of your frontend)
+- setup an asset workflow (css and js)
```
Comment (nephridium) [inline guides/migrate-to-embedded-design.md:14]: @gabrielhase @peyerluk How about having having a default behavior where these assets are placed in the codebase and served directly by the livingdocs-backend? That way setup will be easier and no custom workflow will be needed for these in many cases.

Comment (gabrielhase) [reply guides/migrate-to-embedded-design.md:14]: Sure that is the plan. But we have limited resources ;) For now you need to make a custom asset workflow.

**Comment by disufi** [comment]
@gabrielhase design assets workflow documentation added, I tried to cover all aspects but not going to far into details, just in the required topics, let me know in case you think something is missing. Thanks!

File: reference-docs/project-config/directives/html.md [hunk 2]
```diff
@@ -29,4 +29,4 @@ The automatic validation is hardcoded for a set of given embeds and is recognize
 - `iframe` -> only allow iframe embeds
 - `tweet` -> only allow twitter embeds
 
-Use of the client-side embeds (html directive) is not recommended. We advise you to use includes (TODO link) and write your own micro-service to control the rendering of third-party embeds.
\ No newline at end of file
+Use of the client-side embeds (html directive) is not recommended. We advise you to use [includes](./inlude.md) and write your own micro-service to control the rendering of third-party embeds.
```
Comment (marcbachmann) [inline reference-docs/project-config/directives/html.md:32]: ~`inlude.md`~
`include.md`

**Pull request merged**